Dockerfile.base

```diff
@@ -67,8 +67,9 @@ ENV PUPPETEER_SKIP_DOWNLOAD=false
 ENV PLAYWRIGHT_SKIP_DOWNLOAD=false
 ENV DISPLAY=:99
 
-RUN apt update \
-	&& apt install -y --no-install-recommends \
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		awscli \
```

Comment:
Explanation: I've decided to install `awscli` in the base `ci-e2e` image. 

The difference in size is as follows:
```registry.a8c.com/calypso/ci-e2e   latest          22db100e6787   11 hours ago   5.43GB
registry.a8c.com/calypso/ci-e2e   kpi-dashboard   c8c6ba47b813   2 weeks ago    5.68GB
```